Updates the RFC process (#26)
amundsen-io/rfcs: README.md

1. Copy the appropriate template file to `rfcs/000-<rfcName>.md`. Don't assign an RFC number yet, as this is going to be the PR number, and we'll rename the file accordingly if the RFC is accepted.
1. Fill in the RFC. Put care into the details: RFCs that do not present convincing motivation, demonstrate a lack of understanding of the design's impact, or are disingenuous about the drawbacks or alternatives tend to be poorly-received.
1. Submit a pull request. As a pull request, the RFC will receive design feedback from the larger community, and the author should be prepared to revise it in response.

> 1. An RFC Champion will be assigned to the PR, who will be the owner from the maintainers' side, and responsible for representing it in the maintainers team meetings.

Reviewer: That meeting was something we were supposed to hold every 6 months to set up the roadmap, but never got scheduled. I still think it could be useful.

> 1. An RFC Champion will be assigned to the PR, who will be the owner from the maintainers' side, and responsible for representing it in the maintainers team meetings.

Author: I meant this to be a biannual meeting indeed, as we should try to have those meetings to discuss the roadmap, but also the RFCs, and the overall status of the project. 
But I think would be nice to add the reference to community meetings as well.

1. Build consensus and integrate feedback, especially among the [sub-team](sub-teams) responsible. RFCs with broad support are much more likely to make progress than those that don't receive any comments.
1. RFCs rarely go through this process unchanged, especially as alternatives and drawbacks are shown. You can make edits, big and small, to the RFC to clarify or change the design, but make changes as new commits to the pull request, and leave a comment on the pull request explaining your changes. Specifically, do not squash or rebase commits after they are visible on the pull request.
1. RFC must be approved by at least one maintainer, with no -1 on it, in order to start working on it.

> 1. If consensus is not reached through discussion on an RFC, then we must gain maintainers' approval in the form of a vote, with a voting time of 48 hours. Min required votes for Approval/Rejection of an RFC will be 80% of the total votes.

Reviewer: I think this should go inside the FCP section, as it is when we should approve or reject it

1. RFCs that are candidates for inclusion in Amundsen will enter a **"Final Comment Period (FCP)"**, along with a disposition for the RFC (merge, close, or postpone).
    - This step is taken **when enough of the tradeoffs have been discussed** so that the maintainers are in a position to make a decision. That does not require consensus amongst all participants in the RFC thread. However, the argument supporting the disposition on the RFC needs to have already been clearly articulated. There should not be a strong consensus against that position outside of the maintainer's group.
    - The FCP period lasts **seven days**, and the beginning of this period will be signaled with a comment and tag on the RFC's pull request.
    - For RFCs with lengthy discussion, the motion to FCP is usually preceded by a summary comment trying to lay out the current state of the discussion and major tradeoffs/points of disagreement.

>     - Maintainers will ping the `#amundsen` channel about the RFC to attract the community's attention.

Reviewer 2: there is #amundsen-rfcs channel. are we still planning to use that one?

Reviewer: In theory yes, but lately we have been skipping the FCP step. Hopefully we will follow the process closely now.

    - An RFC can be modified based upon feedback from the team and community. Significant modifications may trigger a new final comment period.

> 1. An RFC may be rejected by the team after the public discussion has been settled and comments have been made to summarize the rationale for rejection. A member of the maintainer's group should then close the RFCs associated pull request. Also, an RFC author may withdraw their RFC by closing it themselves.

Reviewer: ```suggestion
1. An RFC may be rejected by the team after the public discussion has been settled and comments have been made to summarize the rationale for rejection. The RFC champion group should then close the RFCs associated pull request. Also, an RFC author may withdraw their RFC by closing it themselves.
```

> 1. An RFC may be accepted at the close of its final comment period. A team member will merge the RFCs associated pull request, at which point the RFC will become 'active'.

Reviewer: ```suggestion
1. An RFC may be accepted at the close of its final comment period. The RFC champion will merge the RFCs associated pull request, at which point the RFC will become 'active'.
```

1. Build consensus and integrate feedback, especially among the [sub-team](sub-teams) responsible. RFCs with broad support are much more likely to make progress than those that don't receive any comments.
1. RFCs rarely go through this process unchanged, especially as alternatives and drawbacks are shown. You can make edits, big and small, to the RFC to clarify or change the design, but make changes as new commits to the pull request, and leave a comment on the pull request explaining your changes. Specifically, do not squash or rebase commits after they are visible on the pull request.
1. RFCs that are candidates for inclusion in Amundsen will enter a **"Final Comment Period (FCP)"**, along with a disposition for the RFC (merge, close, or postpone).
    - This step is taken **when enough of the tradeoffs have been discussed** so that the maintainers are in a position to make a decision. That does not require consensus amongst all participants in the RFC thread. However, the argument supporting the disposition on the RFC needs to have already been clearly articulated. There should not be a strong consensus against that position outside of the maintainer's group.

>     - The FCP period lasts **seven days**, and the beginning of this period will be signaled with a comment and tag on the RFC's pull request.

Reviewer 2: could we make it shorter (e.g 48 hours, or 72 hours)? Often times, we forget to add the label and it takes so long to merge the pr.

Reviewer: In reality this is where the conversation is supposed to happen, before it is the RFC creation process.
IMO 48 or 72 hrs wouldn´t allow us to have many back and forths with people located in different timezones

Reviewer 3: important to note that this changeset isn't modifying this length, this just got picked up as part of a whitespace change -- suggest reading this diff with `?w=1`
